<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XKCD &quot;click and drag&quot; viewer</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="GraphicsView" name="output_view">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="dragMode">
          <enum>QGraphicsView::ScrollHandDrag</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Progress:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="processButton">
            <property name="text">
             <string>Reload</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Zoom:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="output_zoom">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>output_zoom</zorder>
      <zorder>label</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_open_image"/>
    <addaction name="actionSaveOutput"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Textual_output"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuPOIs">
    <property name="title">
     <string>&amp;POIs</string>
    </property>
    <addaction name="action_Clear_POIs"/>
    <addaction name="actionSavePOIs"/>
    <addaction name="actionLoadPOIs"/>
    <addaction name="separator"/>
    <addaction name="action_Draw_lines"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menuPOIs"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="SignalingDockWidget" name="textDock">
   <property name="windowTitle">
    <string>Textual output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPlainTextEdit" name="textOutput">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_open_image">
   <property name="text">
    <string>&amp;Open image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Textual_output">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Textual output</string>
   </property>
  </action>
  <action name="action_Properties">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>P&amp;roperties</string>
   </property>
  </action>
  <action name="actionProcessors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Processing functions</string>
   </property>
  </action>
  <action name="actionInput_image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Input image</string>
   </property>
  </action>
  <action name="action_Clear_POIs">
   <property name="text">
    <string>&amp;Clear POIs</string>
   </property>
  </action>
  <action name="actionSaveOutput">
   <property name="text">
    <string>&amp;Save output</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSavePOIs">
   <property name="text">
    <string>&amp;Save POIs</string>
   </property>
  </action>
  <action name="actionLoadPOIs">
   <property name="text">
    <string>&amp;Load POIs</string>
   </property>
  </action>
  <action name="action_Draw_lines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Draw lines</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SignalingDockWidget</class>
   <extends>QDockWidget</extends>
   <header>signalingdockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Textual_output</sender>
   <signal>toggled(bool)</signal>
   <receiver>textDock</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>612</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
